neural_structured_learning/examples/notebooks/adversarial_cnn_transfer_learning_fashionmnist.ipynb

```diff
@@ -0,0 +1,1263 @@
+        "## Train Model"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "8UFG2YmEMKSD"
+      },
+      "source": [
+        "EPOCHS = 100\n",
+        "\n",
+        "es_callback = tf.keras.callbacks.EarlyStopping(monitor='val_loss', \n",
+        "                                               patience=2, \n",
+        "                                               restore_best_weights=False,\n",
+        "                                               verbose=1)\n",
+        "\n",
+        "history = adv_model.fit(train_data, validation_data=val_data,\n",
+        "                        validation_steps=val_steps, \n",
+        "                        batch_size=batch_size,\n",
+        "                        callbacks=[es_callback], \n",
+        "                        epochs=EPOCHS,\n",
+        "                        verbose=1)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-COJiiKj-THJ"
+      },
+      "source": [
+        "## Visualize Learning Curves"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "YNvZC8iIMh7r"
+      },
+      "source": [
+        "fig, ax = plt.subplots(1, 2, figsize=(10, 4))\n",
+        "\n",
+        "history_df = pd.DataFrame(history.history)\n",
+        "history_df[['loss', 'val_loss']].plot(kind='line', \n",
+        "                                      ax=ax[0])\n",
+        "history_df[['sparse_categorical_accuracy', \n",
+        "            'val_sparse_categorical_accuracy']].plot(kind='line', \n",
+        "                                                     ax=ax[1]);"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "q5qBXLpJ-VOa"
+      },
+      "source": [
+        "## VGG-19 Adversarial Model Performance on Organic Test Dataset\n",
+        "\n",
+        "Here we check the performance of our adversarially-trained CNN model on the organic test data (without introducing any perturbations)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Wy-tqqzHNWJV"
+      },
+      "source": [
+        "predictions = adv_model.base_model.predict(test_images_3ch_scaled)\n",
+        "prediction_labels = np.argmax(predictions, axis=1)\n",
+        "print(classification_report(test_labels, prediction_labels, \n",
+        "                            target_names=class_names))\n",
+        "pd.DataFrame(confusion_matrix(test_labels, prediction_labels), \n",
+        "             index=class_names, columns=class_names)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mEqFDKAKy1xU"
+      },
+      "source": [
+        "#### Almost similar performance as our non-adversial trained CNN model!"
```

Comment:
I think this should be simpe 'text cell' and not a separate markdown section.

Reply:
Sure removed the heading